pipelines/zap/pipeline.yml

```diff
@@ -88,9 +88,9 @@ jobs:
       put: slack
       params:
         channel: <%= project['slack_channel'] ? "'##{project['slack_channel']}'" : "{{slack-channel}}" %>
-        icon_url: {{slack-icon}}
```

Comment:
No strong feelings. We probably shouldn't be using CloudApp to host the image for the production pipeline anyway.

https://github.com/18F/concourse-compliance-testing/blob/9bd27272a014eaa4ee08a0810fe43074c97c5ce4/config/prod.example.yml#L13